tests/azure/test_azurecache.py

```
@@ -0,0 +1,185 @@
+    return offset
+
+
+@pytest.fixture(scope="module")
+def cache_lib():
+    """Compile simple_file_cache_test.cc into a shared library."""
+    src = os.path.abspath(SIMPLE_FILE_CACHE_SRC)
+    if not os.path.isfile(src):
+        pytest.skip(f"simple_file_cache_test.cc not found at {src}")
+
+    so_path = os.path.join(tempfile.gettempdir(), "libsimple_file_cache_test.so")
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)